sources/Renderer/OpenGL/Platform/Linux/LinuxGLContextX11.h

```diff
@@ -0,0 +1,100 @@
+/*
+ * LinuxGLContext.h
+ *
+ * Copyright (c) 2025 Lukas Hermanns. All rights reserved.
```

Comment:
Another case of changed copyright notice. Please keep those unchanged except for the filename at the top.